jreleaser.yml

```diff
@@ -73,24 +73,45 @@ release:
         - search: ' \[release\]'
 
 distributions:
-  wave-cli:
-    type: NATIVE_IMAGE
+  wave:
```

Comment:
I would argue the name of the Brew formula should remain `wave-cli`, for consistency with the github repo, which importantly avoids confusion with the `wave` server application.

What do you reckon?

Reply:
ok will test it with wave-cli

Reply:
done